src/App.js

```diff
@@ -1,24 +1,41 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import { Home } from './pages/Home';
+import Details from './pages/Details';
+import Motocycles from './pages/Motocycles';
+import NewMotocycle from './pages/NewMotocycle';
+import ReserveForm from './pages/ReserveForm';
+import Reservation from './pages/Reservations';
+import MainLayout from './layout/main';
+import Delete from './pages/Delete';
+import Login from './pages/auth/login';
+import Register from './pages/auth/register';
+import ForgotPassword from './pages/auth/forgot-password';
+import ResetPassword from './pages/auth/reset-password';
+import Splashscreen from './pages/splashscreen';
 
 function App() {
```

Comment:
- [ ]  The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. Kindly refactor this functional component to ES6 arrow function.

That way, your app will comply to latest standards 💪